respa/settings.py

```diff
@@ -34,6 +34,7 @@
     MAIL_MAILGUN_DOMAIN=(str, ''),
     MAIL_MAILGUN_API=(str, ''),
     RESPA_IMAGE_BASE_URL=(str, ''),
+    VARAAMO_RESOURCES_URL=(str, ''),
```

Comment:
Change this to a more generic name as there can be other clients than Varaamo.

Perhaps `RESPA_ADMIN_VIEW_RESOURCE_URL` ?